src/python/pants/vcs/changed.py

```diff
@@ -148,9 +154,15 @@ class Changed(Subsystem):
         default=None,
         help="Calculate changes contained within a given Git spec (commit range/SHA/ref).",
     )
+    dependents = EnumOption(
```

Comment:
This (and its use above) is the only part of the diff that isn't a straight find-and-replace.